experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/detail.html

```diff
@@ -6,6 +6,39 @@
 {% block title %}{{ experiment.name }}{% endblock %}
 
 {% block main_content %}
+  <!-- Invalid Pages Warnings Card -->
+  {% if experiment.get_invalid_pages %}
+    <div class="alert alert-danger" role="alert">
+      <p class="mb-1">
+        Before this experiment can be reviewed or launched, all required fields must be completed. Fields on the
+        <strong>
+          {% for page in experiment.get_invalid_pages %}
```

Comment:
Ah this will compute it twice, I think you want something like

```py
{% with invalid_pages=experiment.get_invalid_pages %}
  {% if invalid_pages %}
    {% for page in invalid_pages %}
      ...
    {% endfor %}
  {% endif %}
{% endwith %}